packages/autocertifier-client/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "@streamr/autocertifier-client",
+  "version": "0.0.1-tatum.6",
+  "description": "Autocertifier Client for Streamr Network",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/streamr-dev/network.git",
+    "directory": "packages/autocertifier-client"
+  },
+  "main": "dist/src/exports.js",
+  "types": "dist/src/exports.d.ts",
+  "license": "STREAMR NETWORK OPEN SOURCE LICENSE",
+  "author": "Streamr Network AG <contact@streamr.network>",
+  "scripts": {
+    "generate-protoc-code": "./proto.sh",
+    "build": "tsc -b tsconfig.node.json",
+    "check": "tsc -p ./tsconfig.jest.json --noEmit",
+    "clean": "jest --clearCache || true; rm -rf dist *.tsbuildinfo node_modules/.cache || true",
+    "eslint": "eslint --cache --cache-location=node_modules/.cache/.eslintcache/ '*/**/*.{js,ts}'",
+    "prepublishOnly": "npm run clean && NODE_ENV=production tsc -b tsconfig.node.json",
+    "test": "jest test/unit test/integration",
+    "test-unit": "jest test/unit",
+    "test-integration": "jest test/integration",
+    "test-production": "jest test/production",
+    "start": "node dist/src/main.js"
+  },
+  "dependencies": {
+    "@streamr/utils": "0.0.1-tatum.6",
+    "acme-client": "^5.0.0",
+    "body-parser": "^1.20.2",
+    "dns2": "^2.1.0",
+    "eventemitter3": "^5.0.0",
+    "express": "^4.17.1",
+    "heap": "^0.2.6",
+    "node-forge": "^1.3.1",
+    "request": "^2.88.2",
+    "sqlite": "^5.0.1",
+    "sqlite3": "^5.0.3",
+    "uuid": "^9.0.1"
```

Remove unused dependencies (also related type dependencies)

```suggestion
    "eventemitter3": "^5.0.0",
    "node-forge": "^1.3.1",
    "request": "^2.88.2"
```